bench/db_bench.ml

```diff
@@ -113,15 +131,15 @@ module Index = struct
   let write_random () =
     Index_unix.reset_stats ();
     let rw = Index.v ~fresh:true ~log_size (root // "fill_random") in
-    print_results (write rw) nb_entries;
+    print_results (write rw) nb_entries "random_writes";
     write_amplif ();
     rw
 
   let write_seq () =
     Index_unix.reset_stats ();
     let rw = Index.v ~fresh:true ~log_size (root // "fill_seq") in
     Array.sort (fun a b -> String.compare (fst a) (fst b)) random;
-    print_results (write rw) nb_entries;
+    print_results (write rw) nb_entries "sequential_writes";
```

Comment:
We now have 3 strings associated with almost all benchmarks:
 1. the name of the output directory for data (`"fill_seq"` here)
 2. the key of the JSON output data (`"sequential_writes"` here)
 3. the name of the benchmark in the CLI (`"write-keys"` here)

These are often not the same, as in this case, which strikes me as quite confusing. We could unify at least 1 & 2 by passing a `name` string as input to each benchmark. 

3 is not related to this PR, and can be caught later on :slightly_smiling_face: